pkg/openlambda/ol.go

```diff
@@ -100,18 +131,24 @@ func (self *olConfig) Invoke(fName string, args string) (resp *bytes.Buffer, rer
 // of worker PIDs and stuff so we don't have to.
 func (self *olConfig) launchOlWorker() error {
 	self.sessionStarted = true
-	cmd := exec.Command("sudo", "sh", "-c",
-		self.cmd+" worker -d --path="+self.dir)
-	if out, err := cmd.CombinedOutput(); err != nil {
-		return errors.Wrap(err, string(out))
+
+	//Under heavy load (e.g. many goroutines calling Invoke(), we can run out
+	//of host connections. This allows us to support many more concurrent
+	//connections.
+	http.DefaultTransport.(*http.Transport).MaxIdleConnsPerHost = 1024
```

Comment:
Do we care about limiting idle connections? It looks like we could set MaxIdleConns=0 to disable the limit.

Reply:
done.